platform/commonUI/general/src/ui/TreeNodeView.js

```diff
@@ -150,6 +152,11 @@ define([
         return this.li;
     };
 
+    TreeNodeView.prototype.setOpenMct = function (openmct) {
```

Comment:
Let's pass openmct into the constructor, instead of adding a new setter function.